Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Pages/TaskList/SchoolAndTrustInformation/Form7ReceivedDateIntegrationTests.cs

```diff
@@ -0,0 +1,190 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using AutoFixture;
+using Dfe.PrepareConversions.Data.Models;
+using Dfe.PrepareConversions.Tests.Extensions;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.Pages.TaskList.SchoolAndTrustInformation
+{
+   public class Form7ReceivedDateIntegrationTests : BaseIntegrationTests
+   {
+      private static readonly DateTime Tomorrow = DateTime.Today.Add(TimeSpan.FromDays(1));
+      private static readonly DateTime Yesterday = DateTime.Today.Subtract(TimeSpan.FromDays(1));
+      public Form7ReceivedDateIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory) { }
+      private void FillInDateWith(DateTime? date)
```

Comment:
A bit of space between fields, ctor, and helper method would help readability here :)

Reply:
Done

Reply:
Nice 😎